Bottom: 7eda3bdd752c7cc4e36189102817f6b58363b82f
Top:    56cbb0abc9747fb2ca103201fff1eab727ea41fe
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 02:27:04 +0000

configure.ac: Don't copy CFLAGS into CPPFLAGS for X11 tests

CFLAGS should work in its own right, and CPPFLAGS does not need a copy.


---

diff --git a/configure.ac b/configure.ac
index a293e3f..5fe1a81 100644
--- a/configure.ac
+++ b/configure.ac
@@ -704,7 +704,7 @@ for e in $HIDLIST; do
     case $e in
       lesstif )
 	AC_PATH_XTRA
-	CPPFLAGS="$CFLAGS $X_CFLAGS"
+	CPPFLAGS="$X_CFLAGS"
 	AC_CHECK_LIB(X11, XOpenDisplay, , , $X_LIBS)
 	AC_CHECK_LIB(ICE, main, , , $X_LIBS)
 	AC_CHECK_LIB(SM, main, , , $X_LIBS)
